git_machete/cmd.py

```diff
@@ -1449,37 +1474,46 @@ def generate_entries():
             if sha in branch_defs_by_sha_in_reflog:
                 # The practice shows that it's rather unlikely for a given commit to appear on filtered reflogs of two unrelated branches
                 # ("unrelated" as in, not a local branch and its remote counterpart) but we need to handle this case anyway.
-                branch_defs_by_sha_in_reflog[sha] += [branch_def]
+                branch_defs_by_sha_in_reflog[sha] += [branch_def]  # type: ignore
```

Comment:
Yeah, this is likely wrong, comes from missing `List` in `branch_defs_by_sha_in_reflog` type